deployment/deploy_capture/bundle.spec

```diff
@@ -85,67 +85,72 @@ elif platform.system() == 'Linux':
                    name='pupil_capture')
 
 elif platform.system() == 'Windows':
-	import sys, os, os.path
-
-	system_path = os.path.join(os.environ['windir'], 'system32')
-
-	print "Using Environment:"
-	python_path = None
-	package_path = None
-	for path in sys.path:
-		print " -- " + path
-		if path.endswith("scripts"):
-			python_path = os.path.abspath(os.path.join(path, os.path.pardir))
-		elif path.endswith("site-packages"):
-			lib_dir = os.path.abspath(os.path.join(path, os.path.pardir))
-			python_path = os.path.abspath(os.path.join(lib_dir, os.path.pardir))
-			package_path = path
-	if (python_path and package_path):
-		print "PYTHON PATH @ " + python_path
-		print "PACKAGE PATH @ " + package_path
-	else:
-		print "could not find python_path or package_path. EXIT."
-		quit()
-	scipy_imports = ['scipy.integrate']
-	scipy_imports += ['scipy.integrate._ode', 'scipy.integrate.quadrature', 'scipy.integrate.odepack', 'scipy.integrate._odepack', 'scipy.integrate.quadpack', 'scipy.integrate._quadpack']
-	scipy_imports += ['scipy.integrate.vode', 'scipy.integrate.lsoda', 'scipy.integrate._dop', 'scipy.special._ufuncs', 'scipy.special._ufuncs_cxx']
-
-	a = Analysis(['../../pupil_src/capture/main.py'],
-	             pathex=['../../pupil_src/shared_modules/'],
-	             hiddenimports=['pyglui.cygl.shader']+scipy_imports+av_hidden_imports,
-	             hookspath=None,
-	             runtime_hooks=None,
-               excludes=['pyx_compiler','matplotlib'])
-
-
-	pyz = PYZ(a.pure)
-	exe = EXE(pyz,
-	          a.scripts,
-	          exclude_binaries=True,
-	          name='pupil_capture.exe',
-	          icon='pupil-capture.ico',
-	          debug=False,
-	          strip=None,
-	          upx=True,
-	          console=False,
-	          resources=['pupil-capture.ico,ICON,GLFW_ICON'])
-	coll = COLLECT(exe,
-	               a.binaries,
-	               a.zipfiles,
-	               a.datas,
-	               [('glfw3.dll', '../../pupil_src/shared_modules/external/glfw3.dll','BINARY')],
```

Comment:
@pointcontrols - are you proposing a new directory `external` that will contain `.dll`s?